src/components/CreateNewVM.vue

```diff
@@ -324,7 +320,8 @@ async def root():
           use_latest: true
         },
         on: {
-          http: true
+          http: true,
+          persistent: this.newProgram.volumes.length > 0
```

Comment:
This does not make sense to me: the `on.persistent` field defines if a VM is supposed to run all the time (instead of on-demand in response to an HTTP request.

It has nothing to do with volumes.